### ISSUE TITLE
`total` should give the amount of elements without the `limit` constraint

### ISSUE THREAD

**Issue by iamdey** [CaliOpen/Caliopen#32]
## To reproduce:
- require threads with a limit: `http://localhost:4000/api/v1/threads?limit=20&offset=10`
- the response:

```
{
  total: 20,
  threads: [ ...(20 items) ]
}
```
## The expected results:

In order to paginate, the client needs to know the amount of elements available for the given filters, eg.
- `/threads/?limit=20`

```
{
  total: 130,
  threads: [ ...(20 items) ]
}
```
-  `/threads/?limit=20&tag=foo`

```
{
  total: 19,
  threads: [ ... (19 items) ]
}
```

_(the `tag` filter does not exist yet, but I'm sure you get the idea)_

WDYT?

**Comment by iamdey** [comment]
btw it is a regression actually.

**Comment by gdchamal** [comment]
Yes this false total concern only thread as they are not indexed. Will fix when thread structure better define.

**Comment by sapiens-sapide** [comment]
fix by PR #57
